examples/chart/teleport/values.yaml

```diff
@@ -204,3 +206,21 @@ serviceAccount:
   # The name of the ServiceAccount to use.
   # If not set and create is true, a name is generated using the fullname template
   name:
+
+persistence:
+  enabled: true
```

Comment:
let's disable by default, as not everyone has volumes hanging around for example usages

Reply:
updated